other-guides/env-files.md

```diff
@@ -1,16 +1,18 @@
 # `.env` Files and Environment Variables
 
-`.env` files are a type of file that holds environment variables of an application. Environment variables allow you to easily integrate your bot with various online platforms (ex. Heroku), easily split your production and development environment as well as keep important information like your bot token, API tokens and database details secure.
+A `.env` file is a type of file that holds environment variables of an application. Environment variables allow you to easily integrate your bot with various online platforms (ex. Heroku), easily split your production and development environment as well as keep important information like your bot token, API tokens and database details secure.
```

Comment:
Pretty sure that should start with `An`, not `A`

Reply:
You read it as **dot env**.

Reply:
Yeah, I think it would sound weird from a developer stance reading it as "An .env file".